Tests/ProfilerTest.php

```diff
@@ -62,7 +64,12 @@ public function setUp(): void
         $urlGenerator = $this->getMockBuilder(UrlGeneratorInterface::class)->getMock();
         $urlGenerator->method('generate')->willReturn('');
 
-        $this->twig->addExtension(new CodeExtension('', '', ''));
+        if (class_exists(CodeExtensionV7::class)) {
+            $this->twig->addExtension(new CodeExtensionV7('', '', ''));
+        } else {
+            $this->twig->addExtension(new CodeExtension('', '', ''));
+        }
```

Comment:
Let's alias the old extension as `CodeExtensionLegacy` and the new one as `CodeExtension`. This will keep the diff smaller when we drop support for Symfony 5 one fine day.

Reply:
:heavy_check_mark: